lib/link_preview/content.rb

```diff
@@ -311,11 +312,10 @@ def content_type_flash?
     end
 
     def content_type_embed?
-      get_property(:html)
+      get_property(:html) ? true : false
```

Comment:
cast this to property to a boolean - I thought `!!get_property(:html)` was pretty clean but rubocop disagreed